From 438173058f0715159a8df9328f76b4f9be4e63be Mon Sep 17 00:00:00 2001
From: Sunil Goutham <sgoutham@marvell.com>
Date: Fri, 19 Mar 2021 17:02:21 +0530
Subject: [PATCH 1328/1921] CI: Updated CI file to get start tag of 5.4.105
 version

Change-Id: Ia6f95207d6cd5d996663c2278ae433ba09744f90
Signed-off-by: Sunil Goutham <sgoutham@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/48117
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
Tested-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[WK: The original patch got from Marvell sdk11.21.09]
Signed-off-by: Wenlin Kang <wenlin.kang@windriver.com>
---
 scripts/ci/baseline.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/ci/baseline.txt b/scripts/ci/baseline.txt
index 07f1e64ad1e3..c894b4d340e1 100644
--- a/scripts/ci/baseline.txt
+++ b/scripts/ci/baseline.txt
@@ -1,7 +1,7 @@
 #
 # SPDX-License-Identifier:           GPL-2.0
 # https://spdx.org/licenses
-# Copyright (C) 2018 Marvell International Ltd.
+# Copyright (C) 2021 Marvell International Ltd.
 #
 # commit id of base tag for this branch
-b300b28b78145b832f1112d77035111e35112cec (tag: v5.4.74) Linux 5.4.74
+ce615a08404c821bcb3c6f358b8f34307bfe30c9 (tag: v5.4.105) Linux 5.4.105
-- 
2.31.1

